src/kirby/components/toolbar/toolbar.component.ts

```diff
@@ -19,6 +19,8 @@ export class ToolbarComponent implements OnInit {
   @Input() hideBackButton: boolean;
 
   @Output() back: EventEmitter<null> = new EventEmitter<null>();
+  @Output() primarySelected: EventEmitter<null> = new EventEmitter<null>();
```

Comment:
Remove obsolete generic type:
```suggestion
  @Output() primarySelected: EventEmitter = new EventEmitter();
```